### ISSUE TITLE
problem with + function to add layers

### ISSUE THREAD

**Issue by swwilcox** [r-spatial/mapview#311]
Hi, 

I am encountering the same error as described here:
https://stackoverflow.com/questions/60980843/error-using-mapview-package-is-nagetprojectionlsti-argument-is-o.

I attempted loading the development version of mapview as suggested in the link, but it didn't work.

Thanks for the great package!  Hopefully this can be resolved to maximize functionality.

SW

**Comment by tim-salabim** [comment]
Can you please provide a reproducible example?

**Comment by swwilcox** [comment]
Yes, sorry about not addressing that initially.  The only problem is part of the spatial data set I'm working with has privacy restrictions.  Would it be helpful if I copied in output that shows the problem?

**Comment by tim-salabim** [comment]
I'd like to reproduce it on my machine, so you can try to use the build in datasets maybe?

**Comment by swwilcox** [comment]
I can try -- hopefully later today.  Of course with build in datasets it might work fine; could be something quirky with the spatial point file I'm working with (that houses the private data).  Will do my best to get an replicable example as soon as I can.

**Comment by swwilcox** [comment]
In a quick effort, I thought I'd try replicating the examples here:  https://r-spatial.github.io/mapview/articles/articles/mapview_02-advanced.html.

There are a few errors that come up for me in these examples -- not all, but some.  To focus on one example (which mimics what I'm trying to do with my private data) maybe have a look at this:  

# mapview + mapview
mapview(franconia, zcol = "district") + mapview(breweries, zcol = "founded") 

# When I run this line (after running all the preceeding code in the link I copied in above) I get:
Error in if (!is.na(getProjection(lst[[i]]))) { : 
  argument is of length zero

Let me know if this is still insufficient and I can try additional examples.  Thanks!

**Comment by swwilcox** [comment]
Sorry about the large font above.  Text in large font was intended to be commented -- as in R with a # symbol.

**Comment by tim-salabim** [comment]
No worries. I'll look into it soon, probably not before the weekend though

**Comment by swwilcox** [comment]
No worries.  Take good care.  Great job with the package, too!

**Comment by tim-salabim** [comment]
Are you sure you installed with `remotes::install_github("r-spatial/mapview@develop")` ? This runs just fine for me...

**Comment by swwilcox** [comment]
Thanks for the response and for looking into this further.  I am certain that I installed the develop version as you suggested (and as covered in the earlier stackexchange string), and yet I was still encountering a variety of problems.  That said, I reloaded R and R studio today (thinking maybe something more fundamental might be wrong) and tried again with the develop version.  I can happily report that all seems fine!  Not sure what the earlier issue was.  Sorry for the wild goose chase.
